core/src/test/java/org/apache/accumulo/core/metadata/schema/TabletMetadataTest.java

```diff
@@ -108,7 +108,7 @@ public void testAllColumns() {
         .put(le1.getValue());
     LogEntry le2 = new LogEntry("localhost:8020/" + UUID.randomUUID());
     mutation.at().family(LogColumnFamily.NAME).qualifier(le2.getColumnQualifier())
-        .put(le2.getValue());
+        .put(new Value(""));
```

Comment:
Seems like this code could stay the same, is this test failing?

```suggestion
        .put(le2.getValue());
```